src/controllers/account-controller.js

```diff
@@ -1,48 +1,213 @@
 const Accounts = require('../models/accounts');
-const {sendToTelegram} = require('../helpers/telegramBot');
-const {jsonErrors} = require('../helpers/exception-responses');
-const {ResponseREST} = require('../helpers/responses');
+const {StatusCodes} = require ('http-status-codes');
+const mongoose = require('mongoose');
+const {generateCode} = require('../helpers/generateCode');
+const {responseServer, responseNotFound, responseGeneral} = require('../helpers/response-result');
+const {sendEmail} = require('./email-controller');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const {sendEmail} = require("./email-controller");
```